src/WebJobs.Extensions.Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -463,8 +463,8 @@ public override async Task<object> GetValueAsync()
                         {
                             using (var reader = new StreamReader(_request.Body))
                             {
-                                var serializer = JsonSerializer.CreateDefault();
-                                return serializer.Deserialize(reader, typeof(object));
+                                string jsonString = await reader.ReadToEndAsync();
+                                return JsonConvert.DeserializeObject(jsonString);
```

Comment:
This ended up being the actual problem. If you were using `dynamic`, it'd throw in v3. Is this the best way to do this?

Reply:
Looks ok to me